### ISSUE TITLE
README badges

### ISSUE THREAD

**Issue by contolini** [cfpb/capital-framework#165]
I'm adding badges to all the components' repos. Which style do folks prefer?

- flat: ![Build Status](https://img.shields.io/travis/cfpb/cf-buttons.svg) ![Coverage Status](https://img.shields.io/coveralls/cfpb/dash/master.svg)
- flat squared: ![Build Status](https://img.shields.io/travis/cfpb/cf-buttons.svg?style=flat-square) ![Coverage Status](https://img.shields.io/coveralls/cfpb/dash/master.svg?style=flat-square)
- plastic: ![Build Status](https://img.shields.io/travis/cfpb/cf-buttons.svg?style=plastic) ![Coverage Status](https://img.shields.io/coveralls/cfpb/dash/master.svg?style=plastic)

I dig flat squared.

**Comment by Scotchester** [comment]
I vote Flat, but let's make sure the type color contrast is 508-compliant.

**Comment by contolini** [comment]
Welp, it fails with a contrast ratio of 2.12:1

**Comment by contolini** [comment]
JK the drop shadow on flat pushes it to 4.75:1. Going with flat.

**Comment by Scotchester** [comment]
You can customize the colors, too, right?

**Comment by contolini** [comment]
Only when creating custom badges with custom text. There may be other services that let you customize the colors but I'm cool with using flat.

**Comment by Scotchester** [comment]
Ah, okay. I'm cool with that if the shadow makes it compliant.
